RocketChat/Rocket.Chat.ReactNative
# [FIX] Remove Non-null assertion operator in ThreadMessagesView

## Proposed changes

Remove Non-null assertion operator in ThreadMessagesView

## Issue(s)	


## How to test or reproduce


- This PR is dependent from https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3631 and https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3628 to test
- So, you need to go to a room, click on the `Threads button` beside the `Search button` and check if the thread message view is working properly. For a better test, go to a room that is used a lot, as #general in open.rocket.chat

## Screenshots

## Types of changes




## Checklist



## Further comments
